src/service/fixed-layer.js

```diff
@@ -557,9 +536,18 @@ export class FixedLayer {
 
     // Update `top`. This is necessary to adjust position to the viewer's
     // paddingTop.
-    if (state.top && (state.fixed || (state.sticky && !this.transfer_))) {
```

Comment:
Either more explicit or comment above that `state.top` is only truthy when fixed or sticky - it's harder to read spread out state so some overprotection is warranted.